Makefile

```diff
@@ -1,10 +1,10 @@
 ############################# INSTRUCTIONS #############################
 #
 # to compile, run:
-#   make
+#   make [DEBUG=true]
```

Comment:
People might assume that the default is false when you write it like that. I'd probably either invert that example usage, or add a line that states the default value.